### PR TITLE
Slow hands

### PR DESCRIPTION
**Pull request by LassiInkinen**
Tällä ei oo kiire mut avaan tän ny valmiiks, ni voin lisääillä tänne

### CODE REVIEW

File: src/sqlittaaja/__init__.py [hunk 1]
```diff
@@ -15,8 +15,10 @@ def main():
     init_script = config["answer"].get("initialize", "")
     answer = config["answer"]["exercise"]
     exercises = extract(config["exercise"]["path"])
+    # Treshold default 90% if not in config.toml
+    treshold_pct = config["check_options"].get("treshold_pct", 0.9)
```
Comment (kuukkanen) [inline src/sqlittaaja/__init__.py:19]: Remember to validate that `check_options` exists in the config.

**Review by kuukkanen — CHANGES_REQUESTED** [review]

File: src/sqlittaaja/__init__.py (hunk 1 above)
Comment (LassiInkinen) [reply src/sqlittaaja/__init__.py:19]: Forgot to git add config.py

File: src/sqlittaaja/config.py [hunk 2]
```diff
@@ -29,6 +29,8 @@ def validate_config(config: dict[str, Any]):
         raise MissingSectionError("answer")
     if config.get("exercise") is None:
         raise MissingSectionError("exercise")
+    if config.get("check_options") is None:
```
Comment (kuukkanen) [inline src/sqlittaaja/config.py:32]: I'm not sure if this is needed? What if user doesn't specify threshold; they would still need to specify this `check_options` section. You could default to this section being empty.

**Review by kuukkanen — CHANGES_REQUESTED** [review]

File: src/sqlittaaja/config.py (hunk 2 above)
Comment (LassiInkinen) [reply src/sqlittaaja/config.py]: Changed it to default empty and added value validation if it is privided.

**Review by kuukkanen — APPROVED** [review]

**Pull request merged**